src/VolleyManagement.Backend/VolleyManagement.UI/Scripts/VMScripts/TournamentOperations/ScheduleCreateOrEdit.js

```diff
@@ -4,17 +4,21 @@ $(document).ready(function () {
   var currNs = VM.addNamespace("tournament.scheduleEdit"),
     me = {},
     divisionDD;
+  var HomeTeamId = $(".home-team-select :selected").val();
+  var AwayTeamId = $(".away-team-select :selected").val();
 
   me.updateDropdownStates = function () {
     var dropDowns = [
       $(".round-select")[0],
       $(".home-team-select")[0],
       $(".away-team-select")[0]
     ];
-
+   
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3ATrailingWhitespace)

Reply:
it has a view without this line
![image](https://user-images.githubusercontent.com/33187342/38015270-f4c2b5b2-3273-11e8-83f2-ccb2e8b38bb2.png)